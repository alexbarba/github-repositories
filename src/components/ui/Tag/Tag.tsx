import { FC } from 'react'

type Props = {
  className?: string
}

const Tag: FC<Props> = ({ children, className = '' }) => (
  <div
    className={`border border-accent-br rounded-xl w-auto px-2 inline-block text-xs ${className}`}>
    {children}
  </div>
)

export default Tag
