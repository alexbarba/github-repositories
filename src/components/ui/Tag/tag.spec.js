import { render } from '@testing-library/react'
import Tag from './Tag'

describe('Tag', () => {
  it('should be match with snapshot', () => {
    const { asFragment } = render(<Tag>Javascript</Tag>)
    expect(asFragment()).toMatchSnapshot()
  })
})
