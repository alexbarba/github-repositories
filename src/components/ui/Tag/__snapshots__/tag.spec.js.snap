// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag should be match with snapshot 1`] = `
<DocumentFragment>
  <div
    class="border border-accent-br rounded-xl w-auto px-2 inline-block text-xs "
  >
    Javascript
  </div>
</DocumentFragment>
`;
